src/e2e-test/resources/pluginParameters.properties

```diff
@@ -159,6 +159,28 @@ gcsParquetFileSchema=[{"key":"workforce","value":"string"},{"key":"report_year",
   {"key":"race_black","value":"long"},{"key":"race_hispanic_latinx","value":"long"},\
   {"key":"race_native_american","value":"long"},{"key":"race_white","value":"long"},\
   {"key":"tablename","value":"string"}]
+gcsJsonFile=testdata/GCS_JSON_TEST.json
+gcsJsonFileSchema={ "type": "record", "name": "text", "fields": [ { "name":"user", "type": "string" }, { "name": "age", "type": "int" }, { "name": "city", "type": "string" } ] }
+gcsJSONFileOutputSchema={ "type": "record", "name": "text", "fields": [ { "name":"user", "type": "string" }, { "name": "age", "type": "int" }, { "name": "city", "type": "string" } ] }
```

Comment:
I don't think you are using the value anywhere, so remove it.

Reply:
Removed unwanted lines.